src/main/scala/org/broadinstitute/hail/driver/ExportVariantsSolr.scala

```diff
@@ -179,6 +185,24 @@ object ExportVariantsSolr extends Command with Serializable {
         cc
       }
 
+    //delete and re-create the collection
+    try {
+      CollectionAdminRequest.deleteCollection(collection).process(solr)
+      info(s"deleted collection ${collection}")
+    } catch {
+      case e: SolrException => warn(s"exportvariantssolr: unable to delete collection ${collection}: ${e}")
+    }
+
+    try {
+      //zookeeper needs to already have a pre-loaded config named "default_config".
+      //if it doesn't, you can upload it using the solr command-line client:
+      //   solr create_collection -c default_config; solr delete -c default_config
+      CollectionAdminRequest.createCollection(collection, "default_config", numShards, 1).process(solr)
+      info(s"created new collection ${collection}")
+    } catch {
+      case e: SolrException => warn(s"exportvariantssolr: unable to create collection ${collection}: ${e}")
+    }
```

Comment:
I think it's fine if the delete fails since the collection may not have existed before. 
For create, should I change warn(..) => fatal(..)?